app/recurring_job.go

```diff
@@ -876,6 +876,10 @@ func filterVolumesForJob(allowDetached bool, volumes []longhorn.Volume, filterNa
 		if util.Contains(*filterNames, volume.Name) {
 			continue
 		}
+		if volume.Status.RestoreRequired {
+			logger.Debugf("Do not create job for %v volume in restoring the backup", volume.Name)
```

Comment:
Change to `Bypassed to create job for %v volume during restoring from the backup`

Reply:
Modified. Thanks